src/Settings.php

```diff
@@ -346,10 +392,21 @@ public function init() {
 	/**
 	 * Add menu options page.
 	 */
-	public function add_options_page() {
-		$this->options_page = add_options_page( $this->title, $this->title, 'manage_options', $this->key, [ $this, 'admin_page_display' ] );
+	public function add_options_pages() {
+		$this->options_page = add_menu_page( $this->title, $this->title, 'manage_options', $this->key, function () {}, 'dashicons-admin-site-alt' );
+		foreach ( $this->subpages as $path => $subpage ) {
+			add_submenu_page(
+				$this->key,
+				$subpage['title'],
+				$subpage['title'],
+				'manage_options',
+				$path,
+				[ $this, 'admin_page_display' ]
```

Comment:
You can avoid using `global` in `admin_page_display` by changing this callback to 
```php
				$path,
				function () use ( $path ) {
					self::admin_page_display( $path );
				}
			);
```
and changing `admin_page_display()` signature to 
```php
	public function admin_page_display( string $plugin_page ) {
```